avocado/utils/pci.py

```diff
@@ -97,10 +97,11 @@ def get_disks_in_pci_address(pci_address):
     """
     disks_path = "/dev/disk/by-path/"
     disk_list = []
-    for dev in os.listdir(disks_path):
-        if pci_address in dev:
-            link = os.readlink(os.path.join(disks_path, dev))
-            disk_list.append(os.path.abspath(os.path.join(disks_path, link)))
```

Comment:
My only suggestion here to reduce complexity is to invert the logic:
```python
if not os.path.exists(disks_path):
   return disk_list
for dev in ....
```

Reply:
@beraldoleal Thanks